backend/test/auth.test.js

```diff
@@ -0,0 +1,121 @@
+require("dotenv/config");
```

Comment:
this file doesnt use `http-chai` because I am just testing the method directly. This was mostly done for personal practice and learning how to setup auth for the rest of the test cases.